src/main/java/frc/robot/subsystems/NavxStatus.java

```diff
@@ -21,9 +21,9 @@ public NavxStatus() {
     }
 
     public NavxStatus(double forward, double right, double yaw, double pitch) {
-        this.yawAngleDegrees = forward;
-        this.forwardDistance = right;
-        this.rightDistance = yaw;
+        this.yawAngleDegrees = yaw;
+        this.forwardDistance = forward;
+        this.rightDistance = right;
         this.pitchAngleDegrees = pitch;           
```

Comment:
Great catch.  That might explain why the NavX Pose positions didn't make sense while I was driving.  This was caused by the assignments being in a different order than the input parameters, so I'd move the this.yawAngleDegrees = yaw; after the assignment for rightDistance.